extensions/interactions/Continue/directives/continue-validation.service.ts

```diff
@@ -44,12 +45,25 @@ export class ContinueValidationService {
     this.baseInteractionValidationServiceInstance.requireCustomizationArguments(
       customizationArgs, ['buttonText']);
 
+    var textSpecs = InteractionSpecsConstants.INTERACTION_SPECS.Continue;
+    var customizationArgSpecs = textSpecs.customization_arg_specs;
+    var buttonTextMaxLength = (
+      customizationArgSpecs[0].schema.validators[0].max_value);
+
     if (customizationArgs.buttonText.value.length === 0) {
       warningsList.push({
         type: AppConstants.WARNING_TYPES.CRITICAL,
         message: 'The button text should not be empty.'
       });
     }
+
+    if (customizationArgs.buttonText.value.length > buttonTextMaxLength) {
+      warningsList.push({
+        type: AppConstants.WARNING_TYPES.CRITICAL,
+        message: 'The button text should not be too long.'
```

Comment:
`'The button text shouldn't be longer than 50 characters.'`

Reply:
Done!